### DISCUSSION TITLE
PHP Fatal Error after upgrade to self-service-password-1.5.3-1.

### DISCUSSION THREAD

**Discussion by cryansharp** [ltb-project/self-service-password#794] [Q&A] [answered]
I upgraded php to version 7.4.33 on a CentOS 7.9 server and confirmed that self-service-password-1.3-1 was working fine.  I then upgraded self-service-password to version 1.5.3-1 and modified the nginx conf file to point to **"root /usr/share/self-service-password/htdocs"**.  However, I'm now getting a **HTTP ERROR 500 (This page isn't working)** error and the following errors in the nginx error log.

2023/08/10 20:46:02 [error] 1183#1183: *1 FastCGI sent in stderr: "PHP message: PHP Warning:  Use of undefined constant SMARTY - assumed 'SMARTY' (this will throw an Error in a future version of PHP) in /usr/share/self-service-password/htdocs/index.php on line 179PHP message: PHP Warning:  require_once(SMARTY): failed to open stream: No such file or directory in /usr/share/self-service-password/htdocs/index.php on line 179PHP message: **PHP Fatal error:  require_once(): Failed opening required 'SMARTY' (include_path='.:/usr/share/pear:/usr/share/php') in /usr/share/self-service-password/htdocs/index.php on line 179" while reading response header from upstream, client: 10.240.1.87,** server: rva-vm-ssp-01.ironnet.io, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.socket:", host: "rva-vm-ssp-01.ironnet.io"

Am I just missing a permission or config setting since it was a large upgrade?  Let me know if additional information is needed.  Thanks.

**Comment by cryansharp** [comment]
This is fixed.  Didn't realize that the smarty config from the new config.inc.php needed to be copied to the local file.

**Reply by coudot** [reply, marked as answer]
The best is to remove the SMARTY param from local file, unless you want to use a custom Smarty installation.

You should keep in local file only parameters that differ from default configuration.

**Reply by cryansharp** [reply]
Great.  Thank you for that clarification.  I'll make sure that it's in the correct file.
